plugins/source/hubspot/docs/_configuration.md

```diff
@@ -1,16 +0,0 @@
-```yaml copy
```

Comment:
Can we keep this file? We use it for managed syncs (or the example from `overview.md`) but we don't want to show `backend_options` in managed syncs since it's not supported yet

Reply:
I can revert the file, but how do you want to document `backend_options` for local syncs?

Reply:
Keep your changes to `overview.md`, those will be use in https://hub.cloudquery.io/plugins/source/cloudquery/hubspot/latest/docs (where we hide the `_configuration.md` file)